survey/4-ai.yml

```diff
@@ -0,0 +1,70 @@
+title: AI
+label: ai
+position: 4
+questions:
+  - label: "How often do you use AI tools for Dev (Github Copilot, ChatGPT ...) "
+    required: true
+    choices:
+      - Daily
+      - Occasionally
+      - Tried them but not interested
+      - Never
+
+  - label: "If you are using AI based coding assistants for Dev, How much improvement do you see in your productivity?"
+    choices:
+      - Not noticeable
```

Comment:
I find the X says nothing and people might have different interpretations. I suggest:

- Did not improve my productivity
- Somewhat improved my productivity (5% to 10%)
- Helped me to become more productive (10% to 30%)
- I am now very productive (30% to 50%)
- I am a 10x engineer now (+50%)

Reply:
done